### ISSUE TITLE
Tables

### ISSUE THREAD

**Issue by fraukewiese** [simnh/addressing_challenges_oemof#113]
There should be no tables included in the tex.
I now commented them out in the tex.
Captions have to be put to the end of the tex-file.
Find out how to hand in the tables.

**Comment by simnh** [comment]
So wie ich das verstehe, handelt es sich dabei um Tabellen, die z.B. mit excel erstellt werden, in tex ist das ja bei uns teil des compilierten dokuments, was wir gar nicht rausnehmen können...

**Comment by fraukewiese** [comment]
stimmt, kønnen wir wohl doch einfach wieder einkommentieren und dann das entsprechende usepackage aktivieren

Preparing tables

When preparing tables, please follow the formatting instructions below.

    Tables should be numbered and cited in the text in sequence using Arabic numerals (i.e. Table 1, Table 2 etc.).
    Tables less than one A4 or Letter page in length can be placed in the appropriate location within the manuscript.
    Tables larger than one A4 or Letter page in length can be placed at the end of the document text file. Please cite and indicate where the table should appear at the relevant location in the text file so that the table can be added in the correct place during production.
    Larger datasets, or tables too wide for A4 or Letter landscape page can be uploaded as additional files. Please see [below] for more information.
    Tabular data provided as additional files can be uploaded as an Excel spreadsheet (.xls ) or comma separated values (.csv). Please use the standard file extensions.
    Table titles (max 15 words) should be included above the table, and legends (max 300 words) should be included underneath the table.
    Tables should not be embedded as figures or spreadsheet files, but should be formatted using ‘Table object’ function in your word processing program.
    Color and shading may not be used. Parts of the table can be highlighted using superscript, numbering, lettering, symbols or bold text, the meaning of which should be explained in a table legend.
    Commas should not be used to indicate numerical values.

**Comment by fraukewiese** [comment]
Ok, einkommentiert, scheint doch kein Problem zu sein. Kønnte man nochmal draufkucken bzgl. Formatierung, ginge aber auch so
